framework/src/modules/interoperability/mainchain/commands/recover_message.ts

```diff
@@ -92,9 +92,18 @@ export class RecoverMessageCommand extends BaseInteroperabilityCommand<Mainchain
 			}
 		}
 
-		// Check that the CCMs are still pending. We can check only the first one,
```

Comment:
It seems that this line got accidentally lost?

Reply:
added it back